public/multi.html

```diff
@@ -0,0 +1,499 @@
+          };
+
+          counterIncreaseButton.onclick = () => {
+            doc.update((root) => {
+              root.counter.increase(1);
+            });
+          };
+
+          // 04. Todo example
+          doc.subscribe('$.todos', (event) => {
+            console.log('🟡 todos event', event);
+
+            // if we can distinguish add operation
+            // const newTodo = event.value[0].change.operations[0].value.getValue()
+            // addTodo(text);
+            displayTodos();
+            if (event.type === 'local-change') {
+              todoInput.value = '';
+              todoInput.focus();
+            }
+          });
+
+          function displayTodos() {
+            todoList.innerHTML = '';
+            // doc.getRoot().todos.forEach((todo) => {
+            doc.getValueByPath('$.todos').forEach((todo) => {
+              addTodo(todo);
+            });
+          }
+
+          function addTodo(text) {
+            const newTodo = document.createElement('li');
+            newTodo.classList.add('todoItem');
+            newTodo.setAttribute('draggable', 'true');
+            const todoText = document.createElement('span');
+            todoText.classList.add('itemName');
+            todoText.textContent = text;
+            const deleteButton = document.createElement('button');
+            deleteButton.classList.add('trash');
+            deleteButton.textContent = '🗑';
+            const moveUpButton = document.createElement('button');
+            moveUpButton.classList.add('moveUp');
+            moveUpButton.textContent = '⬆';
+            const moveDownButton = document.createElement('button');
+            moveDownButton.classList.add('moveDown');
+            moveDownButton.textContent = '⬇';
+
+            newTodo.appendChild(moveUpButton);
+            newTodo.appendChild(moveDownButton);
+            newTodo.appendChild(todoText);
+            newTodo.appendChild(deleteButton);
+            todoList.appendChild(newTodo);
+          }
+          function handleAddTodo() {
+            const text = todoInput.value;
+            if (text === '') {
+              todoInput.focus();
+              return;
+            }
+
+            doc.update((root) => {
+              root.todos.push(text);
+            });
+          }
+
+          let dragIndex = -1;
+          let dropIndex = -1;
+          function handleDragStart(i) {
+            dragIndex = i;
+          }
+          function handleDragOver(e, i) {
+            e.preventDefault();
+            dropIndex = i;
+          }
+          function handleDragEnd() {
+            if (dragIndex === dropIndex) return;
+            swapTodo(dragIndex, dropIndex);
+            dragIndex = -1;
+            dropIndex = -1;
+          }
+          function swapTodo(dragIdx, dropIdx) {
+            doc.update((root) => {
+              const dragged = root.todos.splice(dragIdx, 1)[0];
+              root.todos.splice(dropIdx, 0, dragged);
```

Comment:
If swap is not implemented as move operation, I think there may be two duplicated items when concurrent editing.

https://github.com/yorkie-team/yorkie-js-sdk/blob/0afacc021d679758e607a76bd03420a6b30a1bf0/src/document/json/array.ts#L81-L96